fluentlenium-core/src/main/java/org/fluentlenium/configuration/PropertiesConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package org.fluentlenium.configuration;
+
+import java.util.Properties;
+
+/**
+ * {@link ConfigurationProperties} based on {@link Properties} object.
+ *
+ * @see ConfigurationProperties
+ */
+public class PropertiesConfiguration extends AbstractPropertiesConfiguration {
+    private Properties properties;
+
+    public PropertiesConfiguration(Properties properties) {
+        super("", "fluentlenium.");
```

Comment:
fluentlenium. is used in a couple of places, let's create enum for that or static field